### ISSUE TITLE
While performing build: conveyor failed to get: failed to retrieve path for base.sif: lstat /base.sif: no such file or directory

### ISSUE THREAD

**Issue by SamuelGreenrod** [hyoo/alphafold_singularity#3]
Hi there,

I'm trying to build the singularity containers but keep getting an error message when it comes to building alphafold.sif. I have successfully built the base.sif container, however when I then run "singularity build --remote alphafold.sif alphafold.def" I get the error message "While performing build: conveyor failed to get: failed to retrieve path for base.sif: lstat /base.sif: no such file or directory". I can't use the --fakeroot parameter as I'm using a university server and my details are not in the required folder.

Base.sif is in the working directory - could it be that it needs it be in a specific singularity folder instead? Thanks!

**Comment by hyoo** [comment]
`--remote` build in a remote machine, https://sylabs.io/guides/3.8/user-guide/build_a_container.html#remote
Is the `base.sif` available in there?

**Comment by SamuelGreenrod** [comment]
Yep so I'm running it in a directory on Linux that contains base.def, base.sif, and alphafold.def. I've also added the path to the base.def in the alphafold.def file but still no luck! As far as I can tell it should be able to find the file!
